### DISCUSSION TITLE
Too many active webgl contexts

### DISCUSSION THREAD

**Discussion by dbauszus-glx** [maplibre/maplibre-gl-js#3065] [Q&A]
With maplibre-gl v3.3.1 we create a multipage html document.

After 8 maps are added to the document we receive error warnings in regards to the number of active webgl contexts.

The oldest context will be removed.

![image](https://github.com/maplibre/maplibre-gl-js/assets/22201617/9531fba6-44ce-47ad-8d68-fc37a26f7b6e)

Is there a way to make the rendered context inactive instead of removing?

**Comment by HarelM** [comment]
You can get the html canvas from the map and get the webgl context, the rest is up to you and the browser basically...
